release-notes/6.0/releases.json

```diff
@@ -125,7 +125,7 @@
         "vs-version": "17.0.0",
         "vs-mac-version": "8.10",
         "vs-support": "Visual Studio 2022 (v17.0)",
-        "vs-mac-support": "Visual Studio 2019 for Mac (v8.10)",
+        "vs-mac-support": "Visual Studio 2022 (v17.0)",
```

Comment:
This needs to be "Visual Studio 2022 for Mac (v17.0 latest preview)".

Reply:
resolving in new commit.